frontend/game/board/builder.js

```diff
@@ -17,6 +21,16 @@ define(function (require) {
 
                 var oppRow = baseRow.cloneNode(true);
                 oppRows.appendChild(oppRow);
+
+                var newScore = baseScore.cloneNode(true);
+                newScore.setAttribute("rownum", i);
+                newScore.innerText="1";
```

Comment:
why the hard-coded "1" and "2" assigned to `innerText` and `innterHTML`? (and why use a different attribute, anyway?)

Reply:
Whoops...
I used that for testing, and it didn't matter after coding the rest, so I forgot them there.